Source/Android/app/src/main/res/layout/list_item_setting.xml

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
+<RelativeLayout android:id="@+id/root"
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="?android:attr/selectableItemBackground"
     android:clickable="true"
     android:focusable="true"
-    android:minHeight="72dp">
+    android:minHeight="72dp"
```

Comment:
```suggestion
```

We don't need this here given that the layout_height is set to wrap_content (Should also be set in the rtl version)